src/opendr/perception/multimodal_human_centric/dependencies.ini

```
@@ -1,10 +1,14 @@
 [runtime]
 # 'python' key expects a value using the Python requirements file format
-#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format 
+#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format
 python=torch==1.9.0
        torchvision==0.10.0
+       protobuf<=3.20.0
        tensorboard>=2.4.1
        tqdm
        imageio>=2.6.0
+       protobuf<=3.20.0
```

protobuf seems to appear twice here.
```suggestion
       protobuf<=3.20.0
       tensorboard>=2.4.1
       tqdm
       imageio>=2.6.0
```

---

Yes indeed, thank you